CONTRIBUTING.rst

```diff
@@ -145,21 +145,20 @@ Clone the repository
     git clone git@github.com:YourLogin/strawberry-sqlalchemy-mapper.git
     cd strawberry-sqlalchemy-mapper
 
-#. You should run::
```

Comment:
We should keep this tutorial too since we don’t want to make it Dev Containers only, it’s just another option. But this tutorial is a default one that we dont use in this project... I mean, we dont use miniconda or venv, we use poetry...
 
 Anyway, what you think about adding something like this and adding a poetry tutorial on this "Manual Setup"? 

```
You have two options to set up your development environment:

Option 1: Using Dev Containers 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

#. Install `Dev Containers`_.

... Tutorial here that you added

Option 2: Manual Setup
~~~~~~~~~~~~~~~~~

You can set up your environment manually if you prefer:

... Tutorial here with a poetry and pre-commit tutorial.
```

What you think about this, @fruitymedley?